benchmarks/collections/hash-dict-vs-blob-dict_get.ts

```diff
@@ -0,0 +1,56 @@
+import { pathToFileURL } from "node:url";
+import { add, complete, configure, cycle, save, suite } from "@typeberry/benchmark/setup.js";
+import { Bytes } from "@typeberry/bytes";
+import { StringHashDictionary } from "@typeberry/collections";
+import { BlobDictionary } from "@typeberry/collections/blob-dictionary.js";
+import { HASH_SIZE, type OpaqueHash } from "@typeberry/hash";
+
+const NO_OF_KEYS = 1000;
+
+function key(n: number) {
+  const key = Bytes.blobFromString(`${n}`);
+  const ret = Bytes.zero(HASH_SIZE);
+  ret.raw.set(key.raw, 0);
+  return ret;
+}
+
+const KEYS = Array.from({ length: NO_OF_KEYS }, (_, i) => key(i));
+
+export default function run() {
+  return suite(
+    "Comparing set operation in two hash dicts",
+
+    add("HashDictionary", () => {
+      const map = StringHashDictionary.new<OpaqueHash, number>();
+      for (let k = 0; k < NO_OF_KEYS; k += 1) {
+        map.set(KEYS[k], k);
+      }
+      return () => {
+        for (let k = 0; k < NO_OF_KEYS; k += 1) {
+          map.get(KEYS[k]);
+        }
+      };
+    }),
+
+    add("BlobDictionary", () => {
+      const map = BlobDictionary.new<OpaqueHash, number>(7);
+      for (let k = 0; k < NO_OF_KEYS; k += 1) {
+        map.set(KEYS[k], k);
+      }
+      return () => {
+        for (let k = 0; k < NO_OF_KEYS; k += 1) {
+          map.get(KEYS[k]);
+        }
+      };
+    }),
+
+    cycle(),
+    complete(),
+    configure({}),
+    ...save(import.meta.filename),
+  );
```

Comment:
Can we add test cases for different list thresholds so that we can clearly see that the best value is chosen?